### DISCUSSION TITLE
Congratulations on the ioSender Release!

### DISCUSSION THREAD

**Discussion by phil-barrett** [terjeio/ioSender#95] [General]
Using it now.  Seems pretty solid so far. Am updating all my machines to use it.

**Comment by terjeio** [comment]
Thanks, took some time. grblHAL documentation next...

**Comment by raenji-sk** [comment]
Nice, great to see you made more progess with it. Switched from bCNC to ioSender for most projects now, works really good. Tested some of the probing functions, simple tool offset and outside diameter, and found noo problems at all. Will update to the latest release ASAP.

**Reply by S2000Stefan** [reply]
I would also like to thank terjeio for the ioSender release.
IoSender is an integral part of my self-mounted milling machine. I like using the programme with its extensive possibilities because it makes my work so much easier and speeds up my workflow.
